video_worker/generate_encode.py

```diff
@@ -113,99 +98,116 @@ def _call(self):
+            return
 
-        if self.EncodeObject.filetype == "mp4":
-            self.ffcommand.append("libx264")
-        elif self.EncodeObject.filetype == "webm":
-            self.ffcommand.append("libvpx")
-        elif self.EncodeObject.filetype == "mp3":
-            self.ffcommand.append("libmp3lame")
+        if self.EncodeObject.filetype == 'mp4':
+            self.ffcommand.append('libx264')
+        elif self.EncodeObject.filetype == 'webm':
+            self.ffcommand.append('libvpx')
+        elif self.EncodeObject.filetype == 'mp3':
+            self.ffcommand.append('libmp3lame')
 
     def _scalar(self):
         if self.ffcommand is None:
-            return None
-        if ENFORCE_TARGET_ASPECT is False:
-            return None
+            return
+        if not ENFORCE_TARGET_ASPECT:
+            return
         if self.EncodeObject.filetype == 'mp3':
-            return None
+            return
+
+        # Padding (if requested and needed)
+        # letter/pillarboxing Command example: -vf pad=720:480:0:38
+        # (target reso, x, y)
 
-        """
-        Padding (if requested and needed)
-        letter/pillarboxing Command example: -vf pad=720:480:0:38
-        (target reso, x, y)
-        """
         horiz_resolution = int(float(self.EncodeObject.resolution) * TARGET_ASPECT_RATIO)
 
-        """BITRATE as int"""
+        # BITRATE as int
         if self.VideoObject.mezz_bitrate != 'Unparsed' and len(self.VideoObject.mezz_bitrate) > 0:
             mezz_parse_bitrate = self.VideoObject.mezz_bitrate.strip().split(' ')[0]
         else:
             mezz_parse_bitrate = None
-        """RESOLUTION as int"""
+
+        # RESOLUTION as int
         if self.VideoObject.mezz_resolution != 'Unparsed' and len(self.VideoObject.mezz_resolution) > 0:
             mezz_vert_resolution = int(self.VideoObject.mezz_resolution.strip().split('x')[1])
             mezz_horiz_resolution = int(self.VideoObject.mezz_resolution.strip().split('x')[0])
         else:
             mezz_vert_resolution = None
             mezz_horiz_resolution = None
-        """Aspect Ratio as float"""
+
+        # Aspect Ratio as float
         if mezz_vert_resolution is not None and mezz_horiz_resolution is not None:
             mezz_aspect_ratio = float(mezz_horiz_resolution) / float(mezz_vert_resolution)
         else:
             mezz_aspect_ratio = None
 
-        """Append commands"""
+        # Append commands
+
+        # TODO-1:
+        # if mezz is None then next condition in or works wich would attempt to check
+        # float(None) technically which is bound to raise TypeError
+        # float() argument must be a string or a number, not 'NoneType'
+        # I believe it should be an `and` condition like below
+        # if mezz_aspect_ratio is not None and float(mezz_aspect_ratio) == float(TARGET_ASPECT_RATIO):
+
         if mezz_aspect_ratio is not None or float(mezz_aspect_ratio) == float(TARGET_ASPECT_RATIO):
```

Comment:
@yro `mezz_aspect_ratio` would be generating `TypeError` if it is set to None. See comments above and below.